app/oxalis/mail/DefaultMails.scala

```diff
@@ -65,6 +65,14 @@ object DefaultMails {
       recipients = List(receiver))
   }
 
+  def resetPasswordMail(name: String, receiver: String, token: String) = {
+    Mail(
+      from = defaultFrom,
+      subject = "Your Oxalis password was reset",
```

Comment:
The name Oxalis is deprecated. Please use webKnossos.

(The same goes for EnvironmentOxalis and SilhouetteOxalis. I suggest naming them WebknossosEnvironment and WebknossosSilhouette – since they are special cases of environment and silhouette, not of oxalis)